vlm/src/main/scala/Implicits.scala

```diff
@@ -13,32 +13,5 @@ object Implicits extends Implicits
 
 
 trait Implicits {
-  implicit def stringToURI(str: String): URI =
```

Comment:
Do you think think that this implicit conversion won't be useful for `vlm`?

Reply:
I think its best to avoid implicit conversions and overloads as much as possible while we're moving things around. They make it much harder to see what is going on and to refactor things.

Reply:
:+1: